en/reference/document-v1-api-reference.html

```diff
@@ -80,9 +80,13 @@ <h2 id="http-requests">HTTP requests</h2>
       <p id="visit">
         Iterate over and get all documents, or a <a href="#selection">selection</a> of documents,
         in chunks, using <a href="#continuation">continuation</a> tokens to track progress.
-        Visits is implemented as a linear scan over the documents in the cluster.
+        Visits are a linear scan over the documents in the cluster.
       </p>
 <pre>
+/document/v1/
```

Comment:
Technically requires a `cluster` parameter, so just substituting this in verbatim from the docs will fail. Include here as well?

Reply:
I considered that, but it's not what is done elsewhere on this page. Perhaps we should do it _all the places_, then?

Reply:
Hmm ... that makes for some very long lines. 
Since this isn't a guide, and you can't copy working examples from it, and also because the mandatory parameters are listed just a few lines further down, I think it's fine as it is?